src/index.ts

```diff
@@ -124,6 +128,13 @@ client.on('messageCreate', async (message) => {
         await stopPlayingClip(audioPlayer);
         // Disconnect from the channel
         disconnectFromChannel(channel);
+      } else if (userCommand === BotCommands.POLL) {
+        // Stop polling if already polling. Otherwise, start polling
+        if (isPolling()) {
+          stopPollingLoLGame();
+        } else {
+          startPollingLoLGame(channel, audioPlayer, PATH_TO_CLIPS);
+        }
```

Comment:
Currently this code block is ignoring `IS_LOL_ANNOUNCER_ENABLED` since it is a manual command; I would think something like `IS_LOL_ANNOUNCER_ENABLED` would only be used to dictate the automatic behavior, but I can add it here if you think otherwise.